### PR TITLE
Update Soteria Medical

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
<details>
<summary>
	About The Pull Request
Screenshoties:
image
This makes the chemistry bigger and moving a few objects

image
Move the storage area down where the patients area is

image
Added two IVs drip to the cryo.

image
Moved the blood closet from the patient area to nearby cryo

image
The patient area is on a location that is more safe and secluded
image
Added curtains to the doctors gearing up area for privacy

image
And modifies this little room so doctors can talk or do inspections or some stuff.

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Why?

**Comment by Guidesu** [comment]
Because chemistry being bigger and using that un-used space is sorta of nicer. And having the medical closet nearby the actual medical so doctors can run to get tools and whatnot is better

**Comment by Guidesu** [comment]
The others are little nitpicks. Always good to improve or whatnot. Thoses are my ideas and suggestion after all. Any errors and things are open to review

**Comment by Guidesu** [comment]
Also praticality

**Comment by cdb-is-not-good** [comment]
why does chem need to be bigger? I'll testmerge this for now to see the full extent of the changes but don't expect it to stay hthat way if i find anything weird.

**Comment by ChefDoggo** [comment]
wasn't the room at the bottom of the PR meant to be another break area? it looks like the one that was moved to inbetween biolab and science that USED to be the little break area in biolab

**Comment by cdb-is-not-good** [comment]
Why exactly were the two chem machines moved so far apart, You realize we often have to use both when research isn't around due to how slowly they recharge?

**Comment by AysheDaArtist** [comment]
That IS the little breakroom, it's just been repurposed for a tiny meeting room.  Hope your patient isn't claustrophobic

I dunno about having a door on the patient room, what if they need to leave?  Is it general access, because I just toss people who don't respond in those beds, and then they wake up and go about their business and leave via the button on the main door

**Comment by Guidesu** [comment]
image

**Comment by Guidesu** [comment]
Fixed the medkits ammount (dum dum)
image
Fixed the lil break area again

image
Added glosses

image
Added doors

Big change was making this exam room into a proper exam room like the old map but bigger with a few tools for RPing inspection and a health scanner for check-ups
image

**Comment by Trilbyspaceclone** [comment]
new exsam room dosnt have lights?

**Comment by Guidesu** [comment]
oh shit you right

**Comment by ChefDoggo** [comment]
> image

the loss of a path from sci to med is horrible, not everyone has chem access and walking all the way around or going under takes longer. i don't see why chem of all places needs this extra space when its generally only used by MAYBE 2 people at a time

**Comment by ChefDoggo** [comment]
TLDR, i think this chem remap is pointless and just worsens the ability to move around SI for no real reason

**Comment by Guidesu** [comment]
> TLDR, i think this chem remap is pointless and just worsens the ability to move around SI for no real reason

I applied your suggestion

**Comment by cdb-is-not-good** [comment]
image
you're missing lockdown shutters. 

image
You still have this door locked and this locker doesnt need to be in here. Also you left the chem fridge in this room and didn't move it with the room.

image
Those consoles should just be where the table is, pixel shifting them north looks like they're out of reach. Also the little waiting area is too scrunched and awkward, why is there a hallway to nowhere? What's that for? Expand it a bit, no 2 wide hallways for a waiting area.

**Comment by AysheDaArtist** [comment]
> the loss of a path from sci to med is horrible, not everyone has chem access and walking all the way around or going under takes longer. i don't see why chem of all places needs this extra space when its generally only used by MAYBE 2 people at a time

Yes please, return this shortcut near staircase.

**Comment by Guidesu** [comment]
Done

**Comment by AysheDaArtist** [comment]
I think the hammer is supposed to be like one of those reflex hammers, but it's just a hammer!

omg I gotta meme this, peak Sojourn

**Comment by Trilbyspaceclone** [comment]
do to not following the requested changes and merge conflicts this is being closed
*Not denined per say but the pr in its current state is not exspectable*
